radish34/docker-compose.yml

```diff
@@ -2,6 +2,12 @@ version: '3.5'
 
 services:
 
+  logger:
```

Comment:
Does this service aggregate the logs too? I'm unclear why it's both a base docker image and also a service.

Reply:
No, this is not a service that aggregates the logs. At first I tried to come up with a solution for log aggregation and those lines are some leftovers. I think this can be addressed in another issue/PR.